src/components/TheHeader.vue

```diff
@@ -14,7 +14,7 @@
           >
         </div>
       </v-toolbar-title>
-      <v-toolbar-items v-if="!smAndDown" class="hidden-xs-only">
+      <v-toolbar-items v-if="!smAndDown" class="hidden-xs">
```

Comment:
What's up with this change? Did a vuetify update change this? Did we have it wrong before?